.github/i18n/README.kr.md

```diff
@@ -406,22 +405,21 @@ Search Pokemon with state-of-the-art visual representation
 
 ## 오픈 거버넌스
 
-[깃허브 이정표](https://github.com/jina-ai/jina/milestones) 지나의 장래의 개선의 길을 내다.
+[깃허브 이정표](https://github.com/jina-ai/jina/milestones)로 Jina의 미래 개선점들에 대한 윤곽을 잡았음
 
-오픈 거버넌스 모델의 일환으로 지나의 거버넌스 모델을 주최한다.[엔지니어링 미팅]((https://hanxiao.io/2020/08/06/Engineering-All-Hands-in-Public/)) 사람들 앞에서 이 확대/축소 회의는 매월 둘째 주 화요일 14:00-15:30(CET)에 반복된다. 모든 사람은 다음 달력 초대를 통해 가입할 수 있다.
+여러분은 우리의 오픈 거버넌스 모델의 일환으로 모두를 위한 Jina의 공학을 주최한다.
+Zoom미팅은 매달 두 번째 화요일마다 진행을 하며 시간은 14:00-15:30(CET)이다. Calendar 초대를 통해 모두 참여가 가능하다.
 
 - [Google 캘린더에 추가](https://calendar.google.com/event?action=TEMPLATE&tmeid=MHIybG03cjAwaXE3ZzRrYmVpaDJyZ2FpZjlfMjAyMDEwMTNUMTIwMDAwWiBjXzF0NW9nZnAyZDQ1djhmaXQ5ODFqMDhtY200QGc&tmsrc=c_1t5ogfp2d45v8fit981j08mcm4%40group.calendar.google.com&scp=ALL)
-- [다운로드 .ics](https://hanxiao.io/2020/08/06/Engineering-All-Hands-in-Public/jina-ai-public.ics)
-
-이 모임은 생방송으로 진행되며 나중에 [유튜브 채널]에 게시될 것이다](https://youtube.com/c/jina-ai).
+- [.ics다운로드 하기](https://hanxiao.io/2020/08/06/Engineering-All-Hands-in-Public/jina-ai-public.ics)
 
+또한 이 회의는 생방송으로 송출될 것이며 이 후에 [유튜브 채널에 영상으로 제작될 것이다.](https://youtube.com/c/jina-ai).
 ## 참여하기
 
-지나(Jina)는 오픈소스 프로젝트다. [우리는 고용하고 있다.](https://jobs.jina.ai) 풀스택 개발자, 전도사, 프로젝트 매니저 등이 오픈 소스의 차세대 신경 검색 생태계를 구축한다.
-
+Jina는 오픈소스 프로젝트이다. 우리는 풀스택 개발자, evangelists, 프로젝트 매니저들을 [채용](https://jobs.jina.ai)하여 뉴럴 탐색 생태계를 오픈소스에 구축하려고 한다.
 
-## 면허증
+## 라이선스
 
-저작권 (c) 2020 지나 AI 유한회사. 모든 권리 보유.
+Copyright (c) 2020 Jina AI Limited. All rights reserved
 
-Jina는 Apache License 버전 2.0에 따라 라이센스를 받는다.[전체 라이센스 텍스트는 LICE를 참조하십시오.](LICENSE)
```

Comment:
I would suggest to keep this part in original English, at least the `Apache License` part.